discord/message.py

```diff
@@ -2492,6 +2500,17 @@ def _handle_call(self, data: CallMessagePayload):
         else:
             self.call = None
 
+    def _handle_soundboard_sounds(self, data: List[SoundboardSoundPayload]):
+        for sound in data:
+            guild = self._state._get_guild(utils._get_as_snowflake(sound, 'guild_id'))
+            try:
+                if guild:
+                    self.soundboard_sounds.append(SoundboardSound(state=self._state, data=sound, guild=guild))
+                else:
```

Comment:
No guild can just mean the bot isn't in the source guild, so this isn't necessarily a default sound. Soundmoji can be sent from any guild the sender is in

Reply:
Ah hm. Not sure what to do about that then since SoundboardSound requires a discord.Guild.